### PR TITLE
Clarify the behavior of delete-k8s-credentials

### PR DESCRIPTION
**Pull request by louischan-oursky**
ref #384

### CODE REVIEW

File: features/270-next-skycli/commands.md [hunk 1]
```diff
@@ -283,7 +283,7 @@ Run kubectl config use-context skygear-skygear-production-myapp to switch to the
 
 `skycli app delete-k8s-credentials`
 
-Delete the credentials remotely if exists. Then delete the config from the machine's kubeconfig.
+Delete the config from the local machine's kubeconfig.
```
Comment (kiootic) [inline features/270-next-skycli/commands.md]: Why not delete the credentials remotely too?

Comment (louischan-oursky) [reply features/270-next-skycli/commands.md]: The TokenRequest is actually a JWT token. [client-go does not expose a high level API to delete TokenRequest.](https://godoc.org/k8s.io/client-go/kubernetes/typed/core/v1#ServiceAccountExpansion)

Comment (kiootic) [reply features/270-next-skycli/commands.md]: I suppose we can delete the service account, and it would have the same effect?

Comment (louischan-oursky) [reply features/270-next-skycli/commands.md]: I have thought about it. See the concerns in the commit message.

**Pull request merged**